__tests__/app.test.js

```diff
@@ -356,3 +356,98 @@ describe('api/articles/:article_id/comments', () => {
     });
   });
 });
+
+describe('api/articles (queries)', () => {
+  describe('GET - Successful Responses', () => {
+    test('status:200 and default sorts the articles by date descending', () => {
+      return request(app)
+        .get('/api/articles')
+        .expect(200)
+        .then(({ body: { articles } }) => {
+          expect(articles).toBeSortedBy('created_at', { descending: true });
+        });
+    });
+    test('status:200 and sorts the articles by author when specified', () => {
+      return request(app)
+        .get('/api/articles?sort_by=author')
+        .expect(200)
+        .then(({ body: { articles } }) => {
+          expect(articles).toBeSortedBy('author', { descending: true });
+        });
+    });
+    test('status:200 and sorts the articles in ascending order when specified', () => {
+      return request(app)
+        .get('/api/articles?sort_by=author&order=ASC')
+        .expect(200)
+        .then(({ body: { articles } }) => {
+          expect(articles).toBeSortedBy('author');
+        });
+    });
+    test('status:200 and filters the articles by specified topic', () => {
+      return request(app)
+        .get('/api/articles?topic=cats')
+        .expect(200)
+        .then(({ body: { articles } }) => {
+          console.log(articles);
+          expect(articles).toHaveLength(1);
+          articles.forEach((article) =>
+            expect(article).toEqual(
+              expect.objectContaining({
+                topic: 'cats',
+              })
+            )
```

Comment:
potentially space to test if a topic is valid but has no articles associated with it - it would be successful just empty

Reply:
Created the test for this underneath the one highlighted